charts/wire-server/requirements.yaml

```diff
@@ -129,3 +129,8 @@ dependencies:
   repository: "file://../integration"
   tags:
     - integration
+- name: mlsstats
+  version: "0.0.42"
+  repository: "file://../mlsstats"
+  tags:
+    - integration
```

Comment:
We shouldn' use the `integration` tag for this because this is not integration tests. 
Let's call the tag `mlsstats`, then edit `charts/wire-server/values.yaml` and disable this by default. Add a comment there saying it is not a good idea because of the reasons we also mentioned in the changelog.